### PR TITLE
Auto-link text in comments

### PR DESCRIPTION
**Pull request by AndrewVos**
image

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -26,6 +26,7 @@ gem 'auto_strip_attributes', '~> 2.0', '>= 2.0.6'
 gem 'httparty', '~> 0.13.7'
 gem 'addressable', '~> 2.3.7'
 gem 'typhoeus', '~> 1.0', '>= 1.0.1'
+gem 'rinku', '~> 1.7', '>= 1.7.3', require: "rails_rinku"
```
Comment (thehenster) [inline Gemfile]: What's with all the gem pinning?

File: spec/features/comments_spec.rb [hunk 2]
```diff
@@ -12,18 +12,35 @@
...
+
+    it "auto links urls in comments" do
+      guide = create(:guide)
+      write_a_comment_successfully(
+        guide: guide,
+        comment: "This is a link: http://google.com",
+      )
+      within ".comments .comment" do
+        expect(page).to have_link "http://google.com", href: "http://google.com"
+      end
```
Comment (thehenster) [inline spec/features/comments_spec.rb]: Nicely written test.

Maybe we could delete the one above? It seems to be covered by this one.

File: Gemfile (hunk 1 above)
Comment (AndrewVos) [inline Gemfile]: This particular gem only patches rails `auto_link` method if you require `rails_rinku`. It literally has no other use to us, so I added it in here.

File: spec/features/comments_spec.rb (hunk 2 above)
Comment (AndrewVos) [inline spec/features/comments_spec.rb]: I agree with you but not 100% sure. I'll look at the code again and have a think.

Comment (AndrewVos) [inline spec/features/comments_spec.rb]: refactored.

**Pull request merged**